app/src/main/java/com/example/mp3demo/data/source/local/LocalDataHandler.java

```diff
@@ -0,0 +1,54 @@
+package com.example.mp3demo.data.source.local;
+
+import android.content.ContentResolver;
+import android.content.Context;
+import android.database.Cursor;
+import android.net.Uri;
+import android.provider.MediaStore;
+
+import com.example.mp3demo.data.model.Audio;
+
+import java.util.ArrayList;
+import java.util.List;
+
+public class LocalDataHandler {
```

Comment:
```suggestion
public class LocalDAO {
```
DAO là Data Access Object